libs/barista-components/filter-field/src/filter-field.ts

```diff
@@ -22,7 +22,7 @@ import {
   useAnimation,
 } from '@angular/animations';
 import { FocusMonitor } from '@angular/cdk/a11y';
-import { coerceBooleanProperty, BooleanInput } from '@angular/cdk/coercion';
+import { BooleanInput, coerceBooleanProperty } from '@angular/cdk/coercion';
```

Comment:
Nitpick.

It would be great if the changeset of the pull request would not include _resorted imports_ as this will make it harder to read. 
In this case I don't mind though.